src/features/admin/idpApiSlice.ts

```diff
@@ -136,15 +137,16 @@ export interface UserIdentityProviders {
 }
 
 export interface IdentityProvider {
-  identityProviderId: string
   alias: string
-  identityProviderTypeId: IDPCategory
   displayName?: string
-  redirectUrl: string
   enabled: boolean
+  identityProviderCategoryId: string
+  identityProviderId: string
+  identityProviderTypeId: IDPCategory
+  redirectUrl: string
   mappers: Array<IDPMapper>
   oidc?: OIDCType
-  saml?: OIDCType
+  saml?: SAMLType
```

Comment:
@nidhigarg-bmw  can you provide me some details why this got changed

Reply:
@jjeroch In swagger this type was different so I managed to changed it accordingly
![image](https://github.com/eclipse-tractusx/portal-frontend/assets/101316912/47fc5a42-91ce-4996-a1a8-6308b74c4050)

Reply:
ok understood, thanks